src/shared/remoteSourceTrackingService.ts

```diff
@@ -52,35 +74,30 @@ export type RemoteSourceTrackingServiceOptions = {
  * This service handles source tracking of metadata between a local project and an org.
  * Source tracking state is persisted to .sfdx/orgs/<orgId>/maxRevision.json.
  * This JSON file keeps track of `SourceMember` objects and the `serverMaxRevisionCounter`,
- * which is the highest `serverRevisionCounter` value of all the tracked elements.
- *
- * Each SourceMember object has 4 fields:
- * * serverRevisionCounter: the current RevisionCounter on the server for this object
- * * lastRetrievedFromServer: the RevisionCounter last retrieved from the server for this object
- * * memberType: the metadata name of the SourceMember
- * * isNameObsolete: `true` if this object has been deleted in the org
+ * which is the highest `RevisionCounter` value of all the tracked elements.
  *
- * ex.
+ * See @MemberRevision for the structure of the `SourceMember` object.  It's SourceMember with the additional lastRetrievedFromServer field
  ```
  {
+    fileVersion: 1,
     serverMaxRevisionCounter: 3,
     sourceMembers: {
-      ApexClass__MyClass: {
-        serverRevisionCounter: 3,
+      ApexClass###MyClass: {
+        RevisionCounter: 3,
```

Comment:
since I added a few more fields from the server, it seemed a good time to use the SourceMember's fields directly instead of a "let's rename them all in the json file"